tests/acceptance/features/apiGraph/activities.feature

```diff
@@ -0,0 +1,101 @@
+Feature: check activities
+  As a user
+  I want to check who made which changes to files
+  So that I can track modifications
+
+
+  Scenario: check activities of a file
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And user "Alice" has uploaded file with content "ownCloud test text file 0" to "/textfile0.txt"
+    When user "Alice" checks the activities for file "textfile0.txt" in space "Personal"
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+      "type": "object",
+      "required": [
+        "value"
+      ],
```

Comment:
try to reduce the schema lines. it won't affect the test but we will have fewer lines in a feature file
```suggestion
      "required": ["value"],
```